extras/dgoss/dgoss

```diff
@@ -89,6 +89,10 @@ case "$1" in
             fi
         fi
         [[ $GOSS_SLEEP ]] && { info "Sleeping for $GOSS_SLEEP"; sleep "$GOSS_SLEEP"; }
+        info "Container health"
+        if ! docker top $id; then
+            docker logs $id
```

Comment:
Should we exit after this?

Otherwise it tries to run tests even after the failure, which can be a bit unclear.

Reply:
I've not included the exit because i didn't want to change the current behaviour but you are right, exit would probably the right decission. Please feel free to add the exit.